src/libcore/atomic.rs

```diff
@@ -1089,3 +1090,23 @@ pub fn fence(order: Ordering) {
         }
     }
 }
+
+macro_rules! impl_Debug {
+    ($($t:ident)*) => ($(
+        #[stable(feature = "atomic_debug", since = "1.3.0")]
+        impl fmt::Debug for $t {
+            fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+                f.debug_tuple(stringify!($t)).field(&self.load(Ordering::Relaxed)).finish()
```

Comment:
I don't have strong felling about this but there's reason for keeping it Relaxed. Why would anybody care about the strictest results if the underlining data can be changing while we build the string output?